packages/params-middleware/src/middleware.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import { RSAA } from 'redux-api-middleware';
+
+import type { Dispatch } from 'redux';
+import type { Action } from './types';
+
+function parametriseEndpoint({ endpoint, types }) {
+  const params = types[0]?.meta?.params;
+
+  if (typeof endpoint === 'function' || !params) {
+    return endpoint;
+  }
+
+  const searchParams = new URLSearchParams();
+
+  Object.keys(params)
+    .filter(key => !!params[key])
```

Comment:
What about value **0**? I think that it will be omitted, we should skip only null and undefined